### ISSUE TITLE
ui_ActionMenu

### ISSUE THREAD

**Issue by Damianu** [Noobulater/gmforge-core#6]
My suggestion is to create ui_ActionMenu that can be embeded to charactersheet, it'd gather all actions from items/character and put them in list view. It'd use "actionMenu" similarly to "hot" for hotbar. Main difference would be that "actionMenu" would determine if that action can be added to that menu not if its present. That's because this menu wouldnt just list actions, instead it'd be collection of rollbuttons with saved parameters for rolls.
So for example your item has "Attack" action. You can go to action menu, press add > items > item you want > Attack, then you can prefill bonuses etc and press save. You can have multiple instances of one action call allowing you to use different prefilled bonuses.
I think it's possible to implement and would allow users to call actions from neat menu and have variations saved.
It would also display "Choices" as dropdown menu. So for example if your bow needs Damage with choice Dexterity you can add that action call to this menu and after 1press it'd always roll that Damage/Dexterity from that item.

**Comment by Noobulater** [comment]
Yes this is definitely a need. Trying to think of the best way to implement it.
